lumber-generate.js

```diff
@@ -30,31 +33,45 @@ program
 
   const config = await new CommandGenerateConfigGetter(program).perform();
   let schema = {};
+
   if (program.db) {
-    const connection = await new Database().connect(config);
-    schema = await new DatabaseAnalyzer(connection, config, true).perform();
+    const connectionPromise = new Database().connect(config);
+    spinners.add('database-connection', { text: 'Connecting to database' }, connectionPromise);
```

Comment:
```suggestion
    spinners.add('database-connection', { text: 'Connecting to your database' }, connectionPromise);
```
if you want to be consistent with "Analyzing your database"

or you can adapt "Analyzing you database" to "Analyzing the database".